<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Miraculous Owonubi, Femi Bankole</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>573</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>xRecog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/logos/xrecog-64px.svg</normaloff>:/logos/xrecog-64px.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout2">
    <item row="1" column="0">
     <widget class="QLabel" name="copyrightLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(180, 180, 180);</string>
      </property>
      <property name="text">
       <string>Copyright (c) 2020 Femi Bankole</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="attendanceTab">
       <attribute name="title">
        <string>Attendance</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout3">
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="tablesHLayout">
          <item>
           <widget class="QGroupBox" name="presentGroup">
            <property name="title">
             <string>Present</string>
            </property>
            <layout class="QGridLayout" name="gridLayout10">
             <item row="0" column="0">
              <widget class="QTableWidget" name="presentTable">
               <property name="sizeAdjustPolicy">
                <enum>QAbstractScrollArea::AdjustIgnored</enum>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="showDropIndicator" stdset="0">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::ExtendedSelection</enum>
               </property>
               <property name="cornerButtonEnabled">
                <bool>false</bool>
               </property>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>Matric Code</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>First Name</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Middle Name</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Last Name</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Year</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Course Of Study</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="absentGroup">
            <property name="title">
             <string>Absent</string>
            </property>
            <layout class="QGridLayout" name="gridLayout8">
             <item row="0" column="0">
              <widget class="QTableWidget" name="absentTable">
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="cornerButtonEnabled">
                <bool>false</bool>
               </property>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>Matric Code</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>First Name</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Middle Name</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Last Name</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Year</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Course Of Study</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="captureHLayout1">
          <item>
           <widget class="QPushButton" name="startCameraButton">
            <property name="text">
             <string>Start Camera</string>
            </property>
            <property name="icon">
             <iconset theme="camera">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopCameraButton">
            <property name="text">
             <string>Stop Camera</string>
            </property>
            <property name="icon">
             <iconset theme="stop">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QFormLayout" name="searchFormLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="searchLabel">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="searchLineEdit">
            <property name="placeholderText">
             <string>Enter a search query: 18-6472, Jamie Alexander, etc...</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <layout class="QHBoxLayout" name="footerHLayout">
          <item>
           <layout class="QFormLayout" name="totalFormLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="totalLabel">
              <property name="text">
               <string>Total</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="totalLineEdit">
              <property name="text">
               <string>0</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="presentFormLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="presentLabel">
              <property name="text">
               <string>Present</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="presentLineEdit">
              <property name="text">
               <string>0</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="absentFormLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="absentLabel">
              <property name="text">
               <string>Absent</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="absentLineEdit">
              <property name="text">
               <string>0</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QToolButton" name="printToolButton">
            <property name="text">
             <string>Print</string>
            </property>
            <property name="icon">
             <iconset theme="document-print">
              <normaloff>.</normaloff>.</iconset>
            </property>
            <property name="popupMode">
             <enum>QToolButton::MenuButtonPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <addaction name="actionPrintPreview"/>
            <addaction name="actionExportCSV"/>
            <addaction name="actionExportHTML"/>
            <addaction name="actionExportMarkdown"/>
            <addaction name="actionShowHTMLReport"/>
            <addaction name="actionShowMarkdownReport"/>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="registerTab">
       <attribute name="title">
        <string>Register</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout12">
        <item row="0" column="1">
         <spacer name="registerSpacer1">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <layout class="QFormLayout" name="registerFormLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="firstNameLabel">
            <property name="text">
             <string>First Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="firstNameLineEdit">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Arya</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="middleNameLabel">
            <property name="text">
             <string>Middle Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="middleNameLineEdit">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Shirley</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lastNameLabel">
            <property name="text">
             <string>Last Name</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lastNameLineEdit">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Stark</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="yearLabel">
            <property name="text">
             <string>Year</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="yearSpinBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="matricNumberLabel">
            <property name="text">
             <string>Matric Number</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="matricNumberLineEdit">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>A8324</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="courseOfStudyLabel">
            <property name="text">
             <string>Course of Study</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="courseComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="currentText">
             <string/>
            </property>
            <property name="currentIndex">
             <number>-1</number>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::InsertAtBottom</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="captureHLayout2">
          <item>
           <spacer name="captureSpacer1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="captureButton">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Capture</string>
            </property>
            <property name="icon">
             <iconset theme="camera">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="captureSpacer2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="5" column="1">
         <spacer name="registerSpacer5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <spacer name="registerSpacer3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1">
         <layout class="QHBoxLayout" name="submitHLayout">
          <item>
           <spacer name="submitSpacer1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="resetButton">
            <property name="text">
             <string>Reset</string>
            </property>
            <property name="icon">
             <iconset theme="undo">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="registerButton">
            <property name="text">
             <string>Register</string>
            </property>
            <property name="icon">
             <iconset theme="user-identity">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="submitSpacer2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <spacer name="registerSpacer2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="2">
         <spacer name="registerSpacer4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionResetAttendance"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionResetAttendance">
   <property name="icon">
    <iconset theme="document-revert">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Reset Attendance</string>
   </property>
  </action>
  <action name="actionPrintPreview">
   <property name="icon">
    <iconset theme="document-print-preview">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Print Preview</string>
   </property>
  </action>
  <action name="actionExportHTML">
   <property name="icon">
    <iconset theme="document-export">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Export HTML</string>
   </property>
  </action>
  <action name="actionExportMarkdown">
   <property name="icon">
    <iconset theme="document-export">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Export Markdown</string>
   </property>
  </action>
  <action name="actionShowHTMLReport">
   <property name="icon">
    <iconset theme="document-preview">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Show HTML Report</string>
   </property>
  </action>
  <action name="actionShowMarkdownReport">
   <property name="icon">
    <iconset theme="document-preview">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Show Markdown Report</string>
   </property>
  </action>
  <action name="actionExportCSV">
   <property name="icon">
    <iconset theme="document-export">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Export CSV</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>startCameraButton</tabstop>
  <tabstop>stopCameraButton</tabstop>
  <tabstop>searchLineEdit</tabstop>
  <tabstop>presentTable</tabstop>
  <tabstop>absentTable</tabstop>
  <tabstop>printToolButton</tabstop>
  <tabstop>totalLineEdit</tabstop>
  <tabstop>absentLineEdit</tabstop>
  <tabstop>presentLineEdit</tabstop>
  <tabstop>firstNameLineEdit</tabstop>
  <tabstop>middleNameLineEdit</tabstop>
  <tabstop>lastNameLineEdit</tabstop>
  <tabstop>yearSpinBox</tabstop>
  <tabstop>matricNumberLineEdit</tabstop>
  <tabstop>courseComboBox</tabstop>
  <tabstop>captureButton</tabstop>
  <tabstop>registerButton</tabstop>
  <tabstop>resetButton</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
